src/client.js

```diff
@@ -8,13 +8,19 @@ var config = require('uiconfig');
 var router = require('router');
 var fetcher = require('fetcher');
 
+require('react-svg-patch');
+
+
 if (config.debug) {
-  localStorage.setItem("debug", "*");
+  localStorage.setItem("debug", "holodex:link");
```

Comment:
yeah nah, we should actually change this to `process.env.DEBUG` so you could have done `DEBUG=holodex:link npm run develop` as we should be doing (i'm starting to learn how to use `debug` properly, hehe).